### PR TITLE
Checkout vfs in an external directory

### PR DESCRIPTION
**Pull request by TheOneRing**
This branch depends on https://github.com/owncloud/client/pull/7393

### CODE REVIEW

File: owncloud/owncloud-client/owncloud-client.py [hunk 1]
```diff
@@ -35,11 +42,15 @@ def __init__(self):
...
 
-        self.subinfo.options.configure.args = "-DUNIT_TESTING=1 "
+        self.subinfo.options.configure.args += "-DUNIT_TESTING=1 "
 
         if 'OWNCLOUD_CMAKE_PARAMETERS' in os.environ:
                 self.subinfo.options.configure.args += os.environ['OWNCLOUD_CMAKE_PARAMETERS']
 
+        if self.subinfo.options.dynamic.useVFSWin:
+            win_vfs = CraftPackageObject.get("owncloud/client-plugin-vfs-win")
+            self.subinfo.options.configure.args += f" -DvfsPlugins={win_vfs.instance.sourceDir()}"
```
Comment (dschmidt) [inline owncloud/owncloud-client/owncloud-client.py]: doesn't this remove the suffix plugin?

File: owncloud/owncloud-client/owncloud-client.py [hunk 2]
```diff
@@ -5,11 +5,14 @@
 import re
 
 class subinfo(info.infoclass):
+    def registerOptions(self):
+        self.options.dynamic.registerOption("useVFSWin", False)
```
Comment (dschmidt) [inline owncloud/owncloud-client/owncloud-client.py]: I'd prefer buildVfsWin - use is very unspecific

File: owncloud/version.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[General]
+name = ownCloud
+branches = master;2.6
+tarballs = 2.5.4.11654
```
Comment (dschmidt) [inline owncloud/version.ini]: What does this mean?

Comment (TheOneRing) [reply owncloud/version.ini]: Thats the last stable tarball release we had and yes our versioning seems to be special ❄️

File: owncloud/owncloud-client/owncloud-client.py (hunk 1 above)
Comment (TheOneRing) [reply owncloud/owncloud-client/owncloud-client.py]: no the cmake code does an append

**Pull request merged**